[2022-08-22 20:10:04,292] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-22T20:05:00+00:00 [queued]>
[2022-08-22 20:10:04,328] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-22T20:05:00+00:00 [queued]>
[2022-08-22 20:10:04,329] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-22 20:10:04,329] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-22 20:10:04,329] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-22 20:10:04,370] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): popula_elastic_finalizados> on 2022-08-22 20:05:00+00:00
[2022-08-22 20:10:04,376] {standard_task_runner.py:52} INFO - Started process 98091 to run task
[2022-08-22 20:10:04,398] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'elastic_desk', 'popula_elastic_finalizados', 'scheduled__2022-08-22T20:05:00+00:00', '--job-id', '1928', '--raw', '--subdir', 'DAGS_FOLDER/etl_elastic_desk.py', '--cfg-path', '/tmp/tmpzc0swb2m', '--error-file', '/tmp/tmp81nwrkbj']
[2022-08-22 20:10:04,399] {standard_task_runner.py:80} INFO - Job 1928: Subtask popula_elastic_finalizados
[2022-08-22 20:10:04,530] {task_command.py:371} INFO - Running <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-22T20:05:00+00:00 [running]> on host 1e1070d9e5ae
[2022-08-22 20:10:04,701] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clecio.antao@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=elastic_desk
AIRFLOW_CTX_TASK_ID=popula_elastic_finalizados
AIRFLOW_CTX_EXECUTION_DATE=2022-08-22T20:05:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-22T20:05:00+00:00
[2022-08-22 20:10:04,716] {base.py:271} INFO - GET http://192.168.2.100:9200/ [status:200 request:0.011s]
[2022-08-22 20:10:08,418] {base.py:271} INFO - HEAD http://192.168.2.100:9200/_template/desk_finalizados [status:200 request:0.004s]
[2022-08-22 20:10:09,169] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/es_pandas/es_pandas.py:58 RuntimeWarning: invalid value encountered in double_scalars
[2022-08-22 20:10:09,680] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.104s]
[2022-08-22 20:10:09,878] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.080s]
[2022-08-22 20:10:10,062] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.138s]
[2022-08-22 20:10:10,247] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.161s]
[2022-08-22 20:10:10,916] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.491s]
[2022-08-22 20:10:11,187] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.209s]
[2022-08-22 20:10:11,586] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.176s]
[2022-08-22 20:10:11,760] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.187s]
[2022-08-22 20:10:12,085] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.349s]
[2022-08-22 20:10:12,170] {logging_mixin.py:115} INFO - 0       22-08-2022
1       22-08-2022
2       22-08-2022
3       22-08-2022
4       22-08-2022
           ...    
4480    23-07-2022
4481    23-07-2022
4482    23-07-2022
4483    23-07-2022
4484    23-07-2022
Name: DataFinalizacao, Length: 4485, dtype: object
[2022-08-22 20:10:12,175] {python.py:173} INFO - Done. Returned value was: None
[2022-08-22 20:10:12,289] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=elastic_desk, task_id=popula_elastic_finalizados, execution_date=20220822T200500, start_date=20220822T201004, end_date=20220822T201012
[2022-08-22 20:10:12,425] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-22 20:10:12,722] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
